src/libraries/System.Numerics.Tensors/tests/TensorTests.cs

```diff
@@ -1169,6 +1171,21 @@ public static void TensorStdDevTests()
             Assert.Equal(0f, Tensor.StdDev(upperLeft));
         }
 
+        [Fact]
+        public static void TensorStdDevComplexTests()
+        {
+            // Test case from issue: Tensor.StdDev vs TensorPrimitivers.StdDev differences for Complex input
```

Comment:
Reference the issue in the comment using its URL: https://github.com/dotnet/runtime/issues/116133